### ISSUE TITLE
[A11y]On 'T' key press in scan mode, Narrator is reading as "table with 0 rows and 0 columns" even though there are no tables

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7891]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069945
Repro Steps:​
1.Start Screen Reader. Hit the URL "https://www.nuget.org" and sign in with valid credentials​
2.Tab till "Sign In" link in the top navigation and press enter​
3.Tab till "Sign in with Microsoft" button and press enter​
4.Select valid user id and press enter​
5.Enter any Username, tab till "Register" button and press enter​
6.Tab till the user menu button and press enter to expand it. Select "Manage Packages" and press enter​
7.Tab till "Published Packages", expand it and select any existing package.(If no packages are available, add a new package)​
8.Tab till "Manage Package" link under "Info" and press enter​
9.Turn on scan mode for Narrator by pressing Caps lock + Space bar
10.Now press 'T' key and check the Narrator behavior ​
​
Actual Result:​
On 'T' key press in scan mode, Narrator is reading as "table with 0 rows and 0 columns" even though there are no tables
​
NOTE: NVDA is working as expected(added attachment for reference)

Expected Result:​
On 'T' key press in scan mode, Narrator should read as "no next table" when there are no tables on the page
​
Notes:
Couldn't get scan mode working without exploding my browser. I am unsure of why a table is being read out. I suspect there is a table somewhere hidden that is being fed into the narrator.

**Comment by ryuyu** [comment]
Unable to reproduce in chrome. No-next-table is read out where expected. Never read out "table with 0 rows and 0 columns"
